streams/src/test/java/org/apache/kafka/streams/state/internals/CachingKeyValueStoreTest.java

```
@@ -215,11 +249,6 @@ public void shouldThrowIfTryingToDeleteFromClosedCachingStore() throws Exception
         store.delete("key");
     }
 
-    @Test
-    public void shouldReturnNullIfKeyIsNull() throws Exception {
```

What is the incentive of this change? I mean, why do you want to throw an exception if `key==null` instead of just returning `null`? It might be a step backwards to introduce more exceptions.

---

This makes the cache store consistent with all other stores that throw an exception on put/get if key is null.